### PR TITLE
Vet clinic database: Add join table for visits

### PR DESCRIPTION
**Pull request by aima98**
### For this project, I worked on the following tasks : 

- [ ] Adding a vets table
- [ ] Creating join tables to handle relationship between tables
- [ ] Adding data to vets, visits and specializations table
- [ ] Adding queries to answer project challenge
- [ ] Screenshot of results : 

join1

join2

join3

join4

join5

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -93,4 +93,67 @@ WHERE owners.full_name = 'Dean Winchester'
...
+FROM animals JOIN visits 
+ON visits.animals_id = animals.id JOIN vets 
+ON vets.id = visits.vet_id WHERE vets.name = 'Maisy Smith' 
+ORDER BY visits.date_of_visit ASC LIMIT 1;
+
+-- Most recent visit: animal information, vet information, and date of visit
+SELECT * FROM animals JOIN visits 
+ON animals.id = visits.animals_id JOIN vets 
+ON vets.id = visits.vet_id 
+ORDER BY visits.date_of_visit DESC;
```
Comment (AbelG101) [inline queries.sql]: - [ ] Ouch! this is a close one, but the query here should return a single record with details for the most recent visit instead you're returning multiple records about visits so please make sure to fix this.

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @aima98 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- DB Schema is properly updated with new tables ✔
- All data is correctly added ✔
- Great query organization ✔
- Correct use of Git Flow ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 2]
```diff
@@ -136,7 +136,8 @@ ON vets.id = visits.vet_id WHERE vets.name = 'Maisy Smith'
 ORDER BY visits.date_of_visit ASC LIMIT 1;
 
 -- Most recent visit: animal information, vet information, and date of visit
-SELECT * FROM animals JOIN visits 
+SELECT animals.name, vets.name, visits.date_of_visit  
+FROM animals JOIN visits 
 ON animals.id = visits.animals_id JOIN vets 
 ON vets.id = visits.vet_id 
 ORDER BY visits.date_of_visit DESC;
```
Comment (TadesseDev) [inline queries.sql]: - [ ] Dear @aima98, I noticed you have made some changes to the returned column/attribute types. But following up on the previous reviewer we need to make sure only a single record/tuple is returned. And for such cases we can use the `LIMIT 1;` selector at the end of the query. 
> Something like...
```sql
 ORDER BY visits. date_of_visit DESC
  LIMIT 1;
```

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @aima98 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights
 - [x] Clean code.🙏


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @aima98 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implementing the requested change. ✔ :+1: 
- The milestone requirements have been met with success. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**